builder/src/Reporting/Exit.hs

```diff
@@ -2426,9 +2439,27 @@ formatToReport problem =
         ]
     FormatBadOutline outline ->
       toOutlineReport outline
-    FormatValidateNotCorrectlyFormatted ->
+    FormatValidateErrors errors ->
       Help.report
         "FILES NOT PROPERLY FORMATTED"
         Nothing
         "The input files were not correctly formatted according to Gren's preferred style."
-        []
+        (mapMaybe validateErrorToDoc $ NonEmptyList.toList errors)
+    FormatErrors errors ->
+      Help.report
+        (show (length errors) <> " FILES CONTAINED ERRORS")
+        Nothing
+        "Some files contained errors and could not be formatted:"
+        (formattingErrorToDoc <$> NonEmptyList.toList errors)
+
+formattingErrorToDoc :: FormattingFailure -> D.Doc
+formattingErrorToDoc formattingError = 
+  case formattingError of
+    FormattingFailureParseError path source err ->
+      Help.syntaxErrorToDoc (Code.toSource source) path err
+
+validateErrorToDoc :: ValidateFailure -> Maybe D.Doc
+validateErrorToDoc validateError =
+  case validateError of
+    VaildateFormattingFailure formattingFailure -> Just $ formattingErrorToDoc formattingFailure
+    ValidateNotCorrectlyFormatted -> Nothing
```

Comment:
In the future we can show a diff here

Reply:
imo, showing a diff goes against the ideal user experience, and lacks a motivating use case.  We don't want people to manually fix what's wrong, or even need to spend time thinking about formatting; we just want them to run `gren format` in normal mode and it will apply the fix.

Reply:
That's a good point!
There still might be use cases like previewing the change, but I can see why we might want to not support it